src/classes/GE_GiftEntryController.cls

```diff
@@ -199,11 +201,10 @@ public with sharing class GE_GiftEntryController {
     @AuraEnabled
     public static DataImport__c upsertDataImport(DataImport__c dataImport) {
         try {
-
             upsert dataImport Id;
 
             return dataImport;
-        } catch (Exception e) {
+        } catch (DmlException e) {
```

Comment:
Revert this change and keep this as generic "Exception" for the time being

Reply:
done